browserApp/__init__.py

```diff
@@ -0,0 +1,35 @@
+
+
+class Foo(object):
+
+    def __init__(self):
+        self.bar = None
+
+    def whatAnimal(self):
+        raise NotImplementedError
+
+    def makeSound(self):
+        animal = self.whatAnimal()
+
+        if animal == "Sheep":
```

Comment:
More types of animals should be implemented.

Reply:
I will do that on the next cycle.